pkg/analytics/service.go

```diff
@@ -0,0 +1,78 @@
+package analytics
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+	"time"
+
+	"github.com/apono-io/apono-cli/pkg/aponoapi"
+	"github.com/apono-io/apono-cli/pkg/clientapi"
+	"github.com/apono-io/apono-cli/pkg/version"
+
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func SendCommandAnalyticsEvent(cmd *cobra.Command, args []string) {
+	endTime := time.Now()
+	startTime, err := GetStartTime(cmd.Context())
+	if err != nil {
+		return
+	}
+
+	client, err := aponoapi.GetClient(cmd.Context())
+	if err != nil {
+		return
+	}
+
+	cmdVersion, err := version.GetVersion(cmd.Context())
+	if err != nil {
+		return
+	}
+
+	commandID, err := GetCommandID(cmd.Context())
+	if err != nil {
+		return
+	}
+
+	properties := map[string]interface{}{
+		commandIDField:       commandID,
+		commandPathField:     cmd.CommandPath(),
+		commandArgsField:     args,
+		cliVersionField:      cmdVersion.Version,
+		operatingSystemField: fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
+		startTimeField:       formatTime(*startTime),
+		endTimeField:         formatTime(endTime),
+		exitCodeField:        0,
+	}
+
+	if shell := os.Getenv("SHELL"); shell != "" {
+		properties[shellField] = shell
+	}
+
+	cmd.Flags().VisitAll(func(flag *pflag.Flag) {
+		if flag.Changed {
+			properties[flagFieldPrefix+flag.Name] = flag.Value
+		}
+	})
+
+	eventName := fmt.Sprintf("Command %s Ran", cmd.CommandPath())
```

Comment:
Command %s Ran -> CLI Command %s
some easier template and easier to understand in the outside system